datasource/keyvaultsecret/test-fixtures/template.pkr.hcl

```diff
@@ -0,0 +1,47 @@
+
+variable "keyvault_id" {
+    type=string
+}
+variable "client_id" {
+    type=string
+}
+variable "client_secret" {
+    type=string
+}
+variable "tenant_id" {
+    type=string
```

Comment:
```suggestion
    type=string
    default = "${env("AZURE_TENANT_ID")}"
```

Reply:
Is TENANT_ID still needed?

It has been removed from the other acceptance test as the ARM_SUBSCRIPTION_ID appears to be enough. But I'm not sure from the context of this change.

Reply:
Actually at first I wanted to stay with subscription Id, but the acceptance test uses azure-sdk-for-go for secret creation/deletion, and as far as I know the SDK only works with tenant Id (https://docs.microsoft.com/en-us/azure/developer/go/azure-sdk-authorization#use-environment-based-authentication). 
For simplicity purpose I also used tenant id in the template.

Reply:
Ah okay I see. That is why `os.Setenv("AZURE_AD_RESOURCE", "https://vault.azure.net")` is also needed. Thanks for the response here TIL.